docs/content/querying/lookups.md

```diff
@@ -296,3 +296,154 @@ To test this setup, you can send key/value pairs to a kafka stream via the follo
 ```
 
 Renames can then be published as `OLD_VAL->NEW_VAL` followed by newline (enter or return)
+
+Dynamic configuration
+---------------------
+
+The following documents the behavior of the cluster-wide config which is accessible through the coordinator.
+The configuration is propogated through the concept of "tier" of servers.
+A "tier" is defined as a group of services which should receive a set of lookups.
+For example, you might have all historicals be part of `__default`, and Peons be part of individual tiers for the datasources they are tasked with.
+
+These configs are accessed using JSON through the following URI template
+
+```
+http://<COORDINATOR_IP>:<PORT>/druid/coordinator/v1/lookups/{tier}/{id}
+```
+
+All URIs below are assumed to have `http://<COORDINATOR_IP>:<PORT>` prepended.
+
+If you have NEVER configured lookups before, you MUST post an empty json object `{}` to `/druid/coordinator/v1/lookups` to initialize the configuration.
+
+These endpoints will return one of the following results:
+
+* 404 if the resource is not found
+* 400 if there is a problem in the formatting of the request
+* 202 if the request was accepted asynchronously (`POST` and `DELETE`)
+* 200 if the request succeeded (`GET` only)
+
+## Configuration propagation behavior
+The configuration is propagated to the historical nodes by the coordinator.
+The historical nodes have an internal API for managing `POST`/`GET`/`DELETE` of lookups.
+The coordinator periodically checks the dynamic configuration for changes and, when it detects a change it does the following:
```

Comment:
Adding timing configs last in this PR because adding config options is small compared to logic changes. Once logic stuff is solidified I'll add config.